xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-test/xwiki-platform-ckeditor-test-docker/src/test/it/org/xwiki/ckeditor/test/ui/LocalizationIT.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.ckeditor.test.ui;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.xwiki.test.docker.junit5.TestReference;
+import org.xwiki.test.docker.junit5.UITest;
+import org.xwiki.test.ui.TestUtils;
+import org.xwiki.ckeditor.test.po.AutocompleteDropdown;
+
+@UITest
+public class LocalizationIT extends AbstractCKEditorIT
```

Comment:
Btw, wouldn't those methods be more relevant in QuickActionIT? Even if they are related to localization, this is first in the context of quick action afaiu?

Reply:
Despite the bug being introduced in 15.5, at the same time as Quick Actions, it affects all XWiki's custom CKEditor plugins within the integration. I use QuickActions to check that the localization module behaves as expected but I could have used the macros plugin for instance.
Now it could fit in QuickActionsIT, I can move it if needed.
Thanks

Reply:
ok, if it's larger then quick action, then I'm ok to keep it a dedicated test class.
In this case it might be interesting  to explicit this in the comments of the class. Currently, it can easily be mistaken for a badly located quick action test :)